pkg/internal/infrastructure/terraform.go

```diff
@@ -74,7 +74,14 @@ func ComputeTerraformerTemplateValues(
 		cloudRouterName   string
 		cN                = map[string]interface{}{
 			"minPortsPerVM":                    int32(2048),
+			"maxPortsPerVM":                    int32(65536),
 			"enableEndpointIndependentMapping": false,
+			"enableDynamicPortAllocation":      false,
+			"icmpIdleTimeoutSec":               int32(30),
```

Comment:
For such cases, casting is not necessary. It doesn't change how they get printed in the yaml.